src/routes/sponsors/+page.svelte

```diff
@@ -39,7 +39,7 @@
         <h5 class="sponsor-msg">Interested in hearing how we can help you? Contact us at&nbsp;</h5>
         <a class="sponsor-msg" href="mailto:ds3@ucsd.edu">ds3@ucsd.edu</a>
         <h5 class="sponsor-msg">or view our sponsor packet&nbsp;</h5>
-        <a class="sponsor-msg" href="/">here</a>
+        <a class="sponsor-msg" href="/DS3_Package.pdf" download = "DS3_Package.pdf">here</a>
```

Comment:
Instead of having the user download the packet if they want to view it, see if you can have it open a new page with the browsers dedicated page. For example, if you click on [this link](https://education.sdsc.edu/training/interactive/201901_running_jobs_on_comet/Webinar-Running-Jobs-on-Comet.pdf), it redirects the page and opens the PDF viewer. This may break a user's experience when using the site.